calcom/cal.com
# feat: Add full names in the description of dynamic events.

## What does this PR do?
Updates description of only dynamic events with full names of the attendees.

Sample screenshots:
* Dynamic events:
With two users:
![image](https://github.com/calcom/cal.com/assets/87374468/b37e933b-37af-4057-a803-496ea05e7d06)

With more than two users: 
![image](https://github.com/calcom/cal.com/assets/87374468/9835434a-9279-4873-8316-0b423adbabb7)

*Normal event:
![image](https://github.com/calcom/cal.com/assets/87374468/2e8be99a-8d8b-4c1b-8a15-89e4940b3004)



Fixes # (issue)
Fixes https://github.com/calcom/cal.com/issues/10194
/claim https://github.com/calcom/cal.com/issues/10194



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change




## How should this be tested?


Create dynamic events with 2 or more attendees, the description on booker should have full names of the attendees, if number of attendees is more than 2 then it should show first two attendees full names with more at the end.
- Are there environment variables that should be set?
- What are the minimal test data to have?
- What is expected (happy path) to have (input and output)?
- Any other important info that could help to test that PR

## Mandatory Tasks


## Checklist



- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
